StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TokenHelper.cs

```diff
@@ -57,6 +57,30 @@ internal static bool IsFirstInLine(this SyntaxToken token, bool allowNonWhitespa
         }
 
         /// <summary>
+        /// Gets a value indicating whether the <paramref name="token"/> is an interpolated
+        /// unary expression.
+        /// </summary>
+        /// <param name="token">The token to process.</param>
+        /// <returns>
+        /// <see langword="true"/> if <paramref name="token"/> is an interpolated unary expression;
+        /// otherwise, <see langword="false"/>.
+        /// </returns>
+        internal static bool IsInterpolatedUnaryExpression(this SyntaxToken token)
```

Comment:
:bulb: It would be better to restrict this to `PrefixUnaryExpressionSyntax` instead of all tokens.

Reply:
➡️ I tried making this change and wasn't thrilled with the result.